specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-04-01/containerInstance.json

```diff
@@ -0,0 +1,1234 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          }
+        }
+      }
+    },
+    "Usage": {
+      "description": "A single usage result",
+      "type": "object",
+      "properties": {
+        "unit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unit of the usage result"
+        },
+        "currentValue": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The current usage of the resource"
+        },
+        "limit": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The maximum permitted usage of the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The name object of the resource",
+          "properties": {
+            "value":{
+              "readOnly": true,
+              "type": "string",
+              "description": "The name of the resource"
+            },
+            "localizedValue": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The localized name of the resource"
+            }
+          }
+        }
+      }
+    },
+    "ContainerGroupListResult": {
+      "description": "The container group list response that contains the container group properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerGroup"
+          },
+          "description": "The list of container groups."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of container groups."
+        }
+      }
+    },
+    "Logs": {
+      "description": "The logs.",
+      "type": "object",
+      "properties": {
+        "content": {
+          "type": "string",
+          "description": "The content of the log."
+        }
+      }
+    },
+    "ContainerExecRequest": {
+      "description": "The start container exec request.",
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "The command to be executed."
+        },
+        "terminalSize": {
+          "type": "object",
+          "description": "The size of the terminal.",
+          "properties": {
+            "row":{
+              "type": "integer",
+              "description": "The row size of the terminal"
+            },
+            "column": {
+              "type": "integer",
+              "description": "The column size of the terminal"
+            }
+          }
+        }
+      }
+    },
+    "ContainerExecResponse": {
+      "description": "The information for the container exec command.",
```

Comment:
Same comment here - the type is used only as server response. Please mark all the properties as read-only.

Reply:
Done